#Header header
#int32 user_id
string[] frame_name
#float32[] confidence
geometry_msgs/Vector3[] position
tg_nao_teleoperation/Eulers[] euler_angle
#geometry_msgs/Quaternion[] orientation